design/authenticators-status/authn-status-new-implem.md

```diff
@@ -0,0 +1,109 @@
+# Implementing Authenticator Status Checks
+The authenticator status check allows operators to check if the authenticator is configured correctly. With this capability, 
+operators can make a RESTful request using the Conjur API and receive a response identifying if configuration was successful.
+If not, the operators receive the proper error identifying the step of failure. 
```

Comment:
The authenticator status check allows a Conjur operator to get an immediate feedback on the authenticator configuration. Once the configuration process is complete, the operator sends a Status request that is responded with the status of the given OIDC authenticator.

If the configuration was successful, the response will indicate that the authenticator is ready for authentication. If not, the response will include the error that prevents the authenticator from authenticating Conjur users.

Reply:
wdyt?

Reply:
This is really good! Merge both versions together. Let me know what you think:
`
The authenticator status check allows operators to check if the authenticator is configured correctly. Once the configuration of an authenticator is complete, 
the operator sends a Status request and the status of the given authenticator is returned. If the configuration was successful, the response received 
will reflect this. If not, the response will include the error that prevents identifying the step of failure.
`

Reply:
Follow up question, do we want to call it the status check? I know Inbal wasnt a fan of that

Reply:
The authenticator status API allows Conjur operators to get an immediate feedback on the authenticator configuration. Once the configuration of an authenticator is complete, the operator sends a Status request and the status of the given authenticator is returned.

If the configuration was successful, the response will reflect this. If not, the response will provide the error identifying the step of failure, supporting the operator to overcome it and configure the authenticator properly.

Reply:
> do we want to call it the status check? I know Inbal wasnt a fan of that

let's go with Status API. please change throughout the page

Reply:
"enable status for authenticators"? or "enable status API for authenticators" or "enable the status webservice for authenticators"?

Reply:
what's the context?